app/controllers/admin/hearings_controller.rb

```diff
@@ -3,7 +3,7 @@
 module Admin
   class HearingsController < Admin::ApplicationController
     before_action :set_prosecution_case, only: %i[new create]
-    before_action :set_hearing, only: %i[show edit update destroy add_plea add_allocation_decision]
+    before_action :set_hearing, only: %i[show edit update destroy add_plea add_allocation_decision add_judicial_result]
```

Comment:
Are we sure we want to automatically create a judicial result every time a hearing is created?